src/style.css

```diff
@@ -0,0 +1,17 @@
+html {
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+}
+
+body,
+h2,
+p {
+  margin: 0;
+}
```

Comment:
For resets better make separete new file, for example reset.css, and there you can write all reset styles:)